cincinnati/src/plugins/internal/cincinnati_graph_fetch.rs

```diff
@@ -143,6 +155,9 @@ impl CincinnatiGraphFetchPlugin {
 #[async_trait]
 impl InternalPlugin for CincinnatiGraphFetchPlugin {
     async fn run_internal(self: &Self, io: InternalIO) -> Fallible<InternalIO> {
+        get_tracer()
+            .get_active_span()
+            .update_name("graph_fetch".to_string());
```

Comment:
Use the predefined constant instead:
```suggestion
            .update_name(Self::PLUGIN_NAME.to_string());
```